src/lib/updating-element.ts

```diff
@@ -755,7 +755,7 @@ export abstract class UpdatingElement extends HTMLElement {
    * }
    * ```
    */
-  protected performUpdate(): void|Promise<unknown> {
+  performUpdate(noUpdatedCallbacks?: boolean): void|Promise<unknown> {
```

Comment:
Maybe consider a default value so you can invert value and the if-check below:

```ts
performUpdate(updatedCallbacks: boolean = true): void|Promise<unknown> {
```

Reply:
Done